docs/deployed-contracts/holesky.md

```diff
@@ -65,12 +63,25 @@ Please be aware that you might encounter unexpected behavior during this period.
 - Gate Seal Factory: [`0x1134F7077055b0B3559BE52AfeF9aA22A0E1eEC2`](https://holesky.etherscan.io/address/0x1134F7077055b0B3559BE52AfeF9aA22A0E1eEC2)
 - Gate Seal: [`0x7f6FA688d4C12a2d51936680b241f3B0F0F9ca60`](https://holesky.etherscan.io/address/0x7f6FA688d4C12a2d51936680b241f3B0F0F9ca60)
 
-### Easy Track
+## Staking modules
+
+- Curated Node Operators: [`0x595F64Ddc3856a3b5Ff4f4CC1d1fb4B46cFd2bAC`](https://holesky.etherscan.io/address/0x595F64Ddc3856a3b5Ff4f4CC1d1fb4B46cFd2bAC)
+- Sandbox: [`0xD6C2ce3BB8bea2832496Ac8b5144819719f343AC`](https://holesky.etherscan.io/address/0xD6C2ce3BB8bea2832496Ac8b5144819719f343AC)
```

Comment:
Maybe we should explain what 'Sandbox' means (it hasn't been announced anywhere, I mean)

Reply:
this is a module for testing, onboarding contributors and all that stuff to try. I think sandbox is quite good naming